app/modbus.mjs

```diff
@@ -242,13 +248,26 @@ export const getDeviceInformation = async (modbusClient) => {
     }
 
     result = await mutex.runExclusive(async () => modbusClient.readHoldingRegisters(597, 3))
+    let model = '-'
```

Comment:
Can we use `unknown` instead here? We use that in other places for (unlikely) default values.

Reply:
Only standalone this far (and with a C++ lib I wrote many years ago  ago - but I never managed to write data to control stuff.
So now I'm making another try.
Had some python stuff that didn't read the data stable enough, but with the small modifications - this seems to work great.
I am running HomeAssistant, so I will try to integrate it there as well - at some point.

Reply:
This project started out as Python but due to data reliability issues I ended up switching to Node.js. The libraries for serial communication are surprisingly unreliable.

Reply:
Yes, that is strange.  So hopefully this works nice and I can control it over mqtt in the future. (and HA)
Have mainly been pulling out metrics as it always before got into a stuck mode where I had to pull the power to get it working again when I tried to write to it.
I have the following, installed 2010:
softwareVersion: 1.97,
modelName: "LTR-6 eco EDE"

Reply:
@JohanElmis check out https://github.com/Jalle19/eda-influxdb if you want metrics

Reply:
@Jalle19 Thanks, I will take a look at it.
Currently bulling out a lot of values, doing some pre-processing looking for changes - or limit to 1 point /5mins. Publishing to MQTT (not the HA one), and storing to InfluxDB with DB writer listening to Mqtt. Dashboards in Grafana.
Getting 70+ electrical values, heating-values, solar power and solar heater the same way.. so it will be some work to change this. (and the naming)